pro/src/service/search/use_case/SearchOutputData.java

```diff
@@ -1,5 +1,19 @@
 package service.search.use_case;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public class SearchOutputData {
+    private final Map<String, Object> result;
```

Comment:
The result returned by DAO is a list of Recipes as we change our design decision